packages/web-react/src/components/Tooltip/UncontrolledTooltip.tsx

```diff
@@ -1,15 +1,16 @@
 import React from 'react';
-import { UncontrolledTooltipProps } from '../../types';
+import { useStyleProps } from '../../hooks';
+import { SpiritTooltipProps } from '../../types';
 import Tooltip from './Tooltip';
 import { useTooltip } from './useTooltip';
 
-export const UncontrolledTooltip = (props: UncontrolledTooltipProps) => {
-  const { children, isDismissible, ...restProps } = props;
-
-  const { open, onClose } = useTooltip({ isDismissible });
+export const UncontrolledTooltip = (props: Omit<SpiritTooltipProps, 'onToggle'>) => {
+  const { children, id } = props;
+  const { isOpen, onToggle } = useTooltip();
+  const { styleProps, props: transferProps } = useStyleProps(props);
```

Comment:
Why there is a need to use `useStyleProps` here? The `UNSAFE_` props will be passed down through transfer props 🤔

Reply:
you are right, there is no need to use it - removed